news/_posts/2020-03-bug-bounty-program.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "Let the hunt begin"
+subtitle: "It's time to make PrestaShop better that it already is!"
+date:   2020-03-11 08:00:00
+authors: [ PierreRambaud, PrestaShop ]
+icon: icon-chat
+image: /assets/images/theme/meta-logo-build.png
+tags: [bug, bounty, program, maintenance]
+---
+
+Protecting the privacy of PrestaShop's users is one of our top priority. We build the software with this goal in mind.
+That's why we decided to welcome help from the outisde through a bounty program to put our security to the test!
+
+# Why this program
+
+Bug bounties are used by many leading companies to improve the security of their products.
+These programs provide an incentive for researchers to responsibly disclose software bugs and allow security teams to
+leverage the external community to help keep users safe.
+
+Cyber criminals from around the world are continuously finding new ways of breaking websites or stealing
+personal information and more if they are related to ecommerce.
+
+By creating this program, and even if it may appear as digging our own grave, it is more about prevention.
+It invites White Hat Hacker to play with us and report issues before a "bad guy" messes up.
+
+# What's in scope
+
+First of all, keep in mind this bounty programm does not concern regular bugs, but only security flaws.
+If you have problems and wish to report a bug, feel free to open an issue on our GitHub repository [PrestaShop/PrestaShop](https://github.com/PrestaShop/PrestaShop/issues/new/choose)
+
+Then, this bounty program covers code from our GitHub repository [PrestaShop/PrestaShop](https://github.com/PrestaShop/PrestaShop).
```

Comment:
is this only for the core? (prestashop/prestashop)
what about the modules that are in other repositories of the organisation?

Reply:
Currently only the Core project and native modules, I need to confirm it again with @toutantic

Reply:
```suggestion
Then, this bounty program covers code from [PrestaShop's core and native modules GitHub repositories](https://github.com/PrestaShop/).
```
or something like that

Reply:
No,  we must keep the main repository only, hunters will see easily that composer is used and only PrestaShop dependencies should be tested

Reply:
Core and native modules yes.
Meaning PrestaShop/PrestaShop plus all PrestaShop/ defined in composer.json